projects/schematics/src/shared/utils/file-utils.ts

```diff
@@ -646,6 +646,12 @@ function removeConstructorParamInternal(
       if (i !== 0) {
         const previousParameter = constructorParameters[i - 1];
         changes.push(new RemoveChange(sourcePath, previousParameter.end, ','));
+        // if removing the first param, cleanup the comma after it
+      } else if (i === 0 && constructorParameters.length > 1) {
+        const commas = findNodes(constructorNode, ts.SyntaxKind.CommaToken);
+        // get the comma that matches the constructor parameter's position
+        const comma = commas[i];
+        changes.push(new RemoveChange(sourcePath, comma.getStart(), ','));
```

Comment:
Just a thought here - if we get desired comma, can we check consecutive SyntaxKind symbol if it's a space so that we can also remove this unnecessary space left after param removal?

Reply:
It gets quite complex when it comes to a case when params are defined in separate rows (which we see often with prettier)